app/retail/templates/landing/contributor/activity.html

```diff
@@ -14,7 +14,7 @@
   You should have received a copy of the GNU Affero General Public License
   along with this program. If not, see <http://www.gnu.org/licenses/>.
 {% endcomment %}
-{% load i18n static  %}
+{% load i18n static %}
```

Comment:
🙌

Reply:
😆 
Wait, I'm trying to squash, but can't due to `ea169ee`.